### PR TITLE
act: 0.2.8 -> 0.2.9

### PR DESCRIPTION
**Pull request by marsam**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/misc/act/default.nix [hunk 1]
```diff
@@ -2,23 +2,24 @@
...
 
-  vendorSha256 = "0ns20vvrj0j921wsx227dxbpga6kll7pxglfqhl53xckrh85yyd8";
+  vendorSha256 = "0qf26g0a2j1mbzlc7xjackww22w9bl1x0iw3q1x6kq7fp8xiwhdn";
 
   buildFlagsArray = [ "-ldflags=-s -w -X main.version=${version}" ];
 
   meta = with lib; {
     description = "Run your GitHub Actions locally";
     homepage = "https://github.com/nektos/act";
+    changelog = "https://github.com/nektos/act/releases/tag/v${version}";
```
Comment (filalex77) [inline pkgs/development/tools/misc/act/default.nix:21]: @marsam Interesting. I saw `meta.changelog` mentioned in the manuals, but seldom saw it being used. Is there any policy/convention for adding this to projects (for examples, ones that have nice changelogs in their release descriptions), or is this just a matter of personal preference?

**Review by Br1ght0ne — APPROVED** [review]
diff LGTM
`nixpkgs-review` passes
executable reports version

File: pkgs/development/tools/misc/act/default.nix (hunk 1 above)
Comment (marsam) [reply pkgs/development/tools/misc/act/default.nix:21]: AFAIK there is no policy, I usually add it to packages which I'm interested in their changelog. 
btw, congrats for becoming a committer, we're delighted to have you!

Comment (bhipple) [reply pkgs/development/tools/misc/act/default.nix:21]: As some extra context, the `r-ryantm` bot will now link to `meta.changelog` if it exists, like so:
https://github.com/NixOS/nixpkgs/pull/89682

So please do add these, as it saves reviewer time and is a nice quality-of-life improvement!

**Pull request merged**